### PR TITLE
Allow max input percent to be specified

### LINKED ISSUE #46: Can't get full speed and full turn with gamepad joystick
My XBox One gamepad has round joysticks, and you can't get to 100% X and 100% Y at the same time, which means you can't get both connected outputs to 100% at the same time.  The maximum you can get is 80% of both.

I have a crude fix for this which is to specify a max input percent, and scale accordingly.  This does mean you lose some precision when using each axis independently.  A better fix would be to take into account the position of the other axis when scaling, but this would be rather more involved.

### PR DESCRIPTION
**Pull request by pdw-mb**
This PR allows a maximum input value to be specified.  This means that an output reaches 100% before the input does.  This is a crude fix for the problem of round gamepad joysticks which don't allow you to get 100% of both axes at the same time.  Fixes #46.

### CODE REVIEW

File: BrickController2/BrickController2/BusinessLogic/PlayLogic.cs [hunk 1]
```diff
@@ -235,6 +235,8 @@ private void SetPreviousOutput(ControllerAction controllerAction, float value)
                 }
             }
 
+            axisValue = axisValue * 100F / controllerAction.MaxInputPercent;
```
Comment (imurvai) [inline BrickController2/BrickController2/BusinessLogic/PlayLogic.cs:238]: I would add an extra check if MaxInputPercent is not zero.

**Comment by imurvai** [comment]
Thanks for this extension. As I don't have an XBox controller I haven't experienced this issue.

File: BrickController2/BrickController2/BusinessLogic/PlayLogic.cs (hunk 1 above)
Comment (pdw-mb) [reply BrickController2/BrickController2/BusinessLogic/PlayLogic.cs]: Good point, now added.

Although I now realise a related issue which is that this field will default to 0, and so will break existing creations, as all inputs will be scaled to nothing.  

I'm not familiar with C# or the SQLite ORM, so I'm not sure how to set a default for the column.  I've manually implemented defaulting in d23e600 but I suspect that there's a cleaner way.

**Comment by pdw-mb** [comment]
> As I don't have an XBox controller I haven't experienced this issue.

Interesting.  What controller do you use?  I've seen other people mentioning this problem but I'm not sure if it's unique to the XBox controller.

With a 10% dead zone to stop motor whine when stopped, and 80% max input so I can get full turn at full speed, there's not a lot of usable range on this one!

**Pull request merged**